### PR TITLE
Support optional dockerfile name

### PR DESCRIPTION
**Pull request by octplane**
- Allow the docker-image module to build Dockerfile which are not named Dockerfile.

### CODE REVIEW

File: cloud/docker/docker_image.py [hunk 1]
```diff
@@ -35,6 +35,12 @@
     required: false
     default: null
     aliases: []
+  dockerfile:
+    description:
```
Comment (bcoca) [inline cloud/docker/docker_image.py:39]: version_added?

File: cloud/docker/docker_image.py [hunk 2]
```diff
@@ -35,6 +35,12 @@
...
+       - Dockerfile to use
+    required: false
+    default: Dockerfile
+    aliases: []
```
Comment (bcoca) [inline cloud/docker/docker_image.py]: no need for aliases

File: cloud/docker/docker_image.py (hunk 1 above)
Comment (octplane) [inline cloud/docker/docker_image.py:39]: I've added 1.9 but I have no clue whether this is correct or not...

Comment (bcoca) [inline cloud/docker/docker_image.py:39]: no, since 1.9 was already released, normally its the 'next version to release', 2.0 in this case

**Comment by octplane** [comment]
Fixed and rebased.

**Comment by octplane** [comment]
Thanks !

**Pull request merged**